src/main/java/com/openlattice/mapstores/TestDataFactory.java

```diff
@@ -154,6 +146,33 @@ public static Principal rolePrincipal() {
         return new Principal( PrincipalType.ROLE, randomAlphanumeric( 5 ) );
     }
 
+    public static SecurablePrincipal securableUserPrincipal() {
+        return securablePrincipal( PrincipalType.USER );
+    }
+
+    public static SecurablePrincipal securablePrincipal( PrincipalType type ) {
+        // TODO after Java 13: use switch expression
+        Principal principal;
```

Comment:
FYI you can make this final in java, unassigned, and if none of the branches of the case set the value the compiler will warn you. Similar to val vs var in Kotlin.